### PR TITLE
feat(claiming): send claiming file to server

### PR DESCRIPTION
**Pull request by Tommytrg**

### CODE REVIEW

File: src/store/wallet.js [hunk 1]
```diff
@@ -920,5 +926,36 @@ export default {
         status: { ...this.state.wallet.status, ...status },
       })
     },
+    async sendClaimingFile(context) {
+      console.log('inside send claimingfile')
```
Comment (gabaldon) [inline src/store/wallet.js:930]: delete console.log

File: src/store/wallet.js [hunk 2]
```diff
@@ -920,5 +926,36 @@ export default {
...
+      const file = new FormData()
+
+      file.append('file', blob, fileName)
+
+      const response = await axios.post(
+        'https://claim.witnet.foundation/upload',
+        file,
+      )
+
+      console.log(response)
```
Comment (gabaldon) [inline src/store/wallet.js:958]: delete console.log

File: src/store/wallet.js [hunk 3]
```diff
@@ -487,6 +491,8 @@ export default {
         key: `${context.rootState.wallet.walletId}_claiming_info`,
         value: claimingFileInfo,
       })
+
+      // sendData()
```
Comment (clbartoli) [inline src/store/wallet.js:495]:
```suggestion
      // sendData()
```

**Review by gabaldon — CHANGES_REQUESTED** [review]

File: src/store/wallet.js (hunk 1 above)
Comment (clbartoli) [inline src/store/wallet.js:930]:
```suggestion
      console.log('inside send claimingfile')
```

File: src/store/wallet.js (hunk 2 above)
Comment (clbartoli) [inline src/store/wallet.js:958]:
```suggestion
      console.log(response)
```

**Review by gabaldon — APPROVED** [review]

**Review by clbartoli — APPROVED** [review]

**Pull request merged**